flow-nodes/api-builder-plugin-format-date/test/test.js

```diff
@@ -35,21 +36,33 @@ describe('flow-node format-date', () => {
 					description: 'Formats a date. See Moment.js format() method.',
 					parameters: {
 						date: {
-							description: 'The date to be formatted.',
-							required: true,
+							description: 'The date to be formatted. Defaults to current date/time. For more information, see https://momentjs.com/docs/#/parsing/string/',
+							required: false,
 							initialType: 'string',
 							schema: {
 								type: 'string',
-								title: 'Date'
+								title: 'Date',
+								example: [ new Date('2020-08-19') ]
 							}
 						},
 						format: {
-							description: 'The desired format. Supports all Moment.js supported formats.',
-							required: true,
+							description: 'Takes a string of tokens and replaces them with their corresponding values. Defaults to ISO-8601 format. For the full list of supported tokens and examples, see https://momentjs.com/docs/#/parsing/special-formats/',
+							required: false,
 							initialType: 'string',
 							schema: {
 								type: 'string',
-								title: 'Format'
+								title: 'Format',
+								default: 'YYYY-MM-DDTHH:mm:ssZ'
+							}
+						},
+						offset: {
+							description: 'The desired UTC Offset in the format ±[hh]:[mm]',
+							required: false,
+							initialType: 'string',
+							schema: {
+								type: 'string',
+								title: 'UTC Offset',
```

Comment:
Ah, gave you the wrong title here - Offset shouldn't be capitalised sorry. (title and description) (UTC offset)